lib/class-wp-rest-dependencies-controller.php

```diff
@@ -0,0 +1,418 @@
+		$handles        = wp_list_pluck( $blocks, $field );
+		$handles        = array_values( $handles );
+		$handles        = array_filter( $handles );
+
+		return $handles;
+	}
+
+	/**
+	 * Retrieves the dependenies schema, conforming to JSON Schema.
+	 *
+	 * @return array Item schema data.
+	 */
+	public function get_item_schema() {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+
+		$schema = array(
+			'$schema'    => 'http://json-schema.org/draft-04/schema#',
+			'title'      => $this->rest_base,
+			'type'       => 'object',
+			'properties' => array(
+				'handle'            => array(
+					'description' => __( 'The handle name.', 'gutenberg' ),
+					'type'        => 'string',
+					'default'     => '',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'src'               => array(
+					'description' => __( 'The handle source.', 'gutenberg' ),
+					'type'        => array( 'boolean', 'string' ),
+					'default'     => false,
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'url'               => array(
+					'description' => __( 'The handle url.', 'gutenberg' ),
+					'type'        => array( 'boolean', 'string' ),
+					'format'      => 'uri',
+					'default'     => false,
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'args'              => array(
+					'description' => __( 'Additional arguments for the handle.', 'gutenberg' ),
+					'type'        => array( 'null', 'integer' ),
+					'default'     => null,
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'ver'               => array(
+					'description' => __( 'The handle version.', 'gutenberg' ),
+					'type'        => array( 'string', 'boolean' ),
+					'default'     => false,
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'extra'             => array(
+					'description'          => __( 'Extra data to supply to the handle.', 'gutenberg' ),
+					'type'                 => 'object',
+					'properties'           => array(),
+					'additionalProperties' => true,
+					'default'              => array(),
+					'context'              => array( 'embed', 'view', 'edit' ),
+					'readonly'             => true,
+				),
+				'textdomain'        => array(
+					'description' => __( 'Translation textdomain set for this dependency.', 'gutenberg' ),
+					'type'        => 'string',
+					'default'     => '',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'translations_path' => array(
+					'description' => __( 'Translation path set for this dependency.', 'gutenberg' ),
+					'type'        => 'string',
+					'default'     => '',
+					'context'     => array( 'embed', 'view', 'edit' ),
+					'readonly'    => true,
+				),
```

Comment:
Would it be possible to add `json_translations` to this, including the default? Otherwise consumers that need the translations have to go through the whole routine of `load_script_textdomain` and also duplicate the default.